packages/manager/modules/vps/src/import/vps.service.js

```diff
@@ -1426,7 +1426,8 @@ export default /* @ngInject */ function VpsService(
       )
       .then((disks) =>
         disks.filter(({ capacity }) => capacity > vpsLinkedDisk.size),
-      );
+      )
+      .catch(() => {});
```

Comment:
**question:** Is it not possible to catch only tech user related failures and send an empty response for this specific case? Having this behavior for all the failures is not a good idea.

Reply:
I don't know the way to take. Because it's child module. It should be a oui message on parent module with the error ?